mgmt/src/models/external/overlay/vpcpeering.rs

```diff
@@ -144,10 +144,21 @@ impl VpcManifest {
         }
     }
     pub fn add_expose(&mut self, expose: VpcExpose) -> ConfigResult {
-        expose.validate()?;
```

Comment:
is it your intention to make it possible to construct invalid configurations?

Why remove the validation check here?

Reply:
This is based on Fredi's feedback from my first version:

- https://github.com/githedgehog/dataplane/pull/445#discussion_r2079843706
- https://github.com/githedgehog/dataplane/pull/445#discussion_r2079899667

The gist is that we prefer having validation in a single place, and that, yes, it's acceptable to build a broken config that we will reject at validation time, if only to show it back to the user.

Reply:
Ok.

I can live with that.

It might be good to have a validated type which wraps this one in that case to express the static requirement that you can't feed an invalid config to the dataplane.

Reply:
Yes, I can look into it as a follow-up.